.github/workflows/monitor.yaml

```diff
@@ -0,0 +1,98 @@
+name: ANT Registration Watchdog
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '*/10 * * * *' # Run every 10 minutes
```

Comment:
reduce to once an hour

Reply:
https://github.com/ar-io/ar-io-ant-registry-process/pull/16/commits/660965dca24ad6611ec101fbd6b3002a20cab000